logging/src/main/java/org/wildfly/swarm/logging/LoggingFraction.java

```diff
@@ -51,6 +60,9 @@
 
     public static final String COLOR_PATTERN = "COLOR_PATTERN";
 
+    public static LoggingFraction loggingFraction;
```

Comment:
To have one singleton / class of the LoggingFraction. Otherwise, we can't chain the methods.